src/commcare_cloud/ansible/group_vars/all.yml

```
@@ -61,9 +61,9 @@ old_s3_blob_db_s3_bucket: ~
 old_s3_bulk_delete_chunksize: ~
 
 # To use these vars without deploying the riak control machine
-# add riakcs_s3_access_key and riakcs_s3_secret_key to the ansible secret config directory
-s3_blob_db_access_key: "{{ riakcs_s3_access_key if riakcs_s3_access_key else (hostvars[riakcs_control]['riak_key'] if s3_blob_db_enabled else '') }}"
-s3_blob_db_secret_key:  "{{ riakcs_s3_secret_key if riakcs_s3_secret_key else (hostvars[riakcs_control]['riak_secret'] if s3_blob_db_enabled else '') }}"
+# add s3_access_key and s3_secret_key to the ansible secret config directory
+s3_blob_db_access_key: "{{ s3_access_key if s3_access_key else (hostvars[riakcs_control]['riak_key'] if s3_blob_db_enabled else '') }}"
```

since we've removed the riak role and related code we can also remove these vars:
```suggestion
s3_blob_db_access_key:  "{{ s3_access_key|default('') }}"
```